models/base/BCEfficiencyFitter.cxx

```diff
@@ -28,29 +28,13 @@
 #include "BCEfficiencyFitter.h"
 
 // ---------------------------------------------------------
-BCEfficiencyFitter::BCEfficiencyFitter()
-    : BCFitter()
-    , fHistogram1(0)
-    , fHistogram2(0)
-    , fFitFunction(0)
-    , fHistogramBinomial(0)
-    , fDataPointType(1)
-{
-    // set default options and values
-    fFlagIntegration = false;
-
-    // set MCMC for marginalization
-    SetMarginalizationMethod(BCIntegrate::kMargMetropolis);
-}
-
-// ---------------------------------------------------------
```

Comment:
The constructor taking a name, since name is defaulted, acts a default constructor